### PR TITLE
Build: optimize JS more

### LINKED ISSUE #161: Optimize or drop JavaScript
I wouldn't call our little scala.js experiment as completely successful: currently every our user have to download the whole 1.2 MiB  of `loglist-scalajs-fastopt.js`. That's not acceptable.

We have to found a way to optimize it dramatically (to be no more than 400 kiB, preferably to 100 kiB or less) or completely rewrite it in plain JavaScript or some lighter language.

### PR DESCRIPTION
**Pull request by ForNeVeR**
This optimizes our JS from 1.2 MiB to just 200 kB.

Closes #161.

### CODE REVIEW

File: scalajvm/app/views/main.scala.html [hunk 1]
```diff
@@ -10,7 +10,7 @@
         <link rel="stylesheet" media="screen" href="@routes.Assets.at("stylesheets/main.css")">
         <link rel="shortcut icon" type="image/png" href="@routes.Assets.at("images/favicon.png")">
         <link rel="alternate" type="application/rss+xml" title="@title" href="@routes.Quotes.getRSSFeed()">
-		<script src="@routes.Assets.at("javascripts/loglist-scalajs-fastopt.js")"></script>
+		<script src="@routes.Assets.at("javascripts/loglist-scalajs-opt.js")"></script>
```
Comment (rexim) [inline scalajvm/app/views/main.scala.html]: So now it's not fast but rather small, right?

Comment (ForNeVeR) [inline scalajvm/app/views/main.scala.html]: Not so fast, but, as you can see, it's 4 bytes smaller: `fastopt` -> `opt`!

Comment (rexim) [inline scalajvm/app/views/main.scala.html]: Oh, it changes everything! Thanks for such a dramatic optimization!

**Comment by rexim** [comment]
LGTM, thanks! :+1:

**Pull request merged**